lib/config.js

```diff
@@ -15,7 +15,7 @@
 var path = require("path"),
     ConfigOps = require("./config/config-ops"),
     ConfigFile = require("./config/config-file"),
-    util = require("./util"),
+    Plugins = require("./config/plugins"),
```

Comment:
This is the way I've been doing these in `./config`. See `ConfigFile`, `ConfigOps`, etc.

I was having a hard time distinguishing between object instances and decided to try and clarify by making them look more like classes with static methods.